index.html

```diff
@@ -0,0 +1,490 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet">
+    <link href="css/styles.css" rel="stylesheet">
+    <title>YouTube Page</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+            <div class="container-fluid">
+                <a href="#" class="navbar-brand"><i data-feather="youtube" class="me-2"></i>YouTube</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>          
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <div class="col-md-5 mx-auto mb-2 mb-lg-0">
+                        <div class="">
+                            <form class="d-flex" role="search">
+                                <div class="input-group">
+                                    <input class="form-control" type="search" placeholder="Search" aria-label="Search">
+                                    <button class="btn btn-outline-success" type="submit" id="button-addon2">
+                                        <i data-feather="search"></i>
+                                    </button>
+                                    <button type="button" class="btn btn-light rounded-circle text-center ms-3">
+                                        <i data-feather="mic" width="15"></i>
+                                    </button>
+                                </div>
+                            </form>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="col-md-1">
+                        <ul class="navbar-nav flex-row flex-wrap ms-md-auto">
+                            <li class="nav-item col-4 mt-1">
+                                <i data-feather="video" color="white"></i>
+                            </li>
+                            <li class="nav-item col-4 mt-1">
+                                <i data-feather="bell" color="white"></i>
+                            </li>
+                            <li class="nav-item col-4 mt-1">
+                                <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle" style="width: 35px; height: 35px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.